2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,38 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
+
+let emptyBottle = 0;
```

Comment:
I love that you are using the variables and reuse them across the functions. These are what we call constant variables, which will never change (their values should not be changed in the code). Therefore, instead of using ``let``, you should use ``const`` to ensure that the values do not accidentally change